src/distilabel/tasks/text_generation/self_instruct.py

```diff
@@ -61,9 +61,17 @@ class SelfInstructTask(TextGenerationTask):
         " You are given a task description and a set of instructions for how to write the prompts for an"
         " specific AI application."
     )
+
     application_description: str = "AI assistant"
     num_instructions: int = 5
 
+    criteria_for_query_generation: str = (
+        "Incorporate a diverse range of verbs, avoiding repetition. "
+        "Ensure queries are compatible with AI model's text generation functions and are limited to 1-2 sentences. "
+        "Design queries to be self-contained and standalone. "
+        """Blend interrogative (e.g., "What is the significance of x?") and imperative (e.g., "Detail the process of x.") styles. """
+    )
+
```

Comment:
```suggestion
        "Incorporate a diverse range of verbs, avoiding repetition.\n"
        "Ensure queries are compatible with AI model's text generation functions and are limited to 1-2 sentences.\n"
        "Design queries to be self-contained and standalone.\n"
        """Blend interrogative (e.g., "What is the significance of x?") and imperative (e.g., "Detail the process of x.") styles.\n"""
    )

```

Reply:
Not really sure of how much it could affect the "\n", but just to keep it it was originally